components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -782,7 +788,7 @@ func NewUpdateProcessingQueue(ctx context.Context, manager *process.StagedManage
 		{
 			stage:     "btp-operator",
 			step:      update.NewApplyReconcilerConfigurationStep(db.Operations(), db.RuntimeStates(), reconcilerClient),
-			condition: update.RequiresReconcilerUpdate,
+			condition: requiresReconcilerUpdate,
```

Comment:
why did you replace the value of the condition in line 757? you can leave the condition as it is and use `disabled` field together

Reply:
the update queue doesn't have `disabled` afaict, only `condition`
https://github.com/kyma-project/control-plane/blob/bf84c20d9f0b9eca0f1e92afae23efe2cc4e2f8c/components/kyma-environment-broker/cmd/broker/main.go#L760-L764